Runtime/SentrySdk.cs

```diff
@@ -146,14 +148,27 @@ private List<Breadcrumb> GetBreadcrumbs()
             _noBreadcrumbs);
     }
 
-    public void OnEnable()
+    public void OnEnabled()
     {
-        Application.logMessageReceived += OnLogMessageReceived;
+        SetDefaultLogAttached(true);
```

Comment:
The word `Attach` can be a bit misleading since we're offering `Attachments` as a feature in Sentry though not on this SDK.

I prefer we add a simple opt-out for capturing errors automatically that can be defined before `OnEnabled` is called.

Since this package is provided as source, I suggest for now you just delete the `OnEnable` method until we can figure out what we do with this. Ideally it would be the same solution we use in the [new SDK](https://github.com/getsentry/sentry-unity)